.prettierignore

```diff
@@ -0,0 +1,6 @@
+*.marko.js
+static/
+dist/
+docs/
+_cdn/
+demo/style.less
```

Comment:
This too? I see that there is some ignore put in demo/style.less but it shows up here.

Reply:
good catch, got that one too